extension/commands/deleteIdentityCommand.ts

```diff
@@ -63,8 +63,8 @@ export async function deleteIdentity(treeItem: IdentityTreeItem): Promise<void>
             return;
         }
 
-        identityName = await UserInputUtil.showIdentitiesQuickPickBox('Choose the identity to delete', identityNames);
-        if (!identityName) {
+        identitiesToDelete = await UserInputUtil.showIdentitiesQuickPickBox('Choose the identities you wish to delete', true, identityNames) as string[];
```

Comment:
Can this just be `Choose the identities to delete`?

Reply:
Changed it